spec/function.dd

```diff
@@ -722,32 +722,30 @@ $(H2 $(LNAME2 property-functions, Property Functions))
 
 $(H2 $(LNAME2 virtual-functions, Virtual Functions))
 
-        $(P Virtual functions are functions that are called indirectly through a
-        function pointer table, called a vtbl[], rather than directly. All
-        $(D public) and $(D protected) member functions which are non-static and
-        are not templatized are virtual unless the compiler can determine that
-        they will never be overridden (e.g. they are marked with $(D final) and
-        do not override any functions in a base class), in which case, it will
-        make them non-virtual. Static or `final` functions with `Objective-C`
-        linkage are virtual as well. This results in fewer bugs caused by not
-        declaring a function virtual and then overriding it anyway.
+        $(P Virtual functions are class member functions that are called indirectly through a
+        function pointer table, called a `vtbl[]`, rather than directly.
         )
 
-        $(P Member functions which are $(D private) or $(D package) are never
-        virtual, and hence cannot be overridden.
+        $(P Member functions that are virtual can be overridden, unless they are `final`.
         )
 
-        $(P Functions with non-D linkage cannot be virtual and hence cannot be
-        overridden.
+        $(P Struct and union member functions are not virtual.
         )
 
-        $(P Member template functions cannot be virtual and hence cannot be
-        overridden.
+        $(P Static member functions are not virtual.
         )
 
-        $(P Functions marked as $(D final) may not be overridden in a
-        derived class, unless they are also $(D private).
-        For example:
+        $(P Member functions which are $(D private) or $(D package) are not virtual.
+        )
+
+        $(P Functions with non-D linkage are not virtual.
```

Comment:
What about C++? 
Or it this the case where the linkage of the function differs from the class.

Reply:
This was copied from the previous description, but is incorrect. Will fix.